monkey/monkey_island/cc/ui/src/components/ui-components/plugins-marketplace/utils.js

```diff
@@ -25,3 +27,50 @@ export const generatePluginId = (plugin) => {
   const type = plugin.type_ || plugin.plugin_type;
   return `${plugin.name}${type}${plugin.version}`;
 }
+
+export const getPluginsGridHeaders = (getRowActions) => [
```

Comment:
`BasePlugins.js` is using the grid. These methods should be moved there instead of utils.